### PR TITLE
feat(config): add Qubino Shades Remote Controller

### LINKED ISSUE #5846: Missing device configuration: Qubino Shades Remote Controller, ZMNKGD1
### Checklist

- [X] It is not in the [configuration DB](https://devices.zwave-js.io/?search=3)

- [X] It was not [merged recently](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Apr+is%3Aclosed) or has a [pending PR](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Aopen+is%3Apr)

### Which device is missing?

Qubino Shades Remote Controller, ZMNKGD1

### Manufacturer ID

0x015A

### Product Type

0x2000

### Product ID

0x0003

### Firmware Version

_No response_

### Is the device listed on the Z-Wave Alliance website?

No

### Do you have a manual?

[Qubino_Shades_Remote.pdf](https://github.com/zwave-js/node-zwave-js/files/11578179/Qubino_Shades_Remote.pdf)

### PR DESCRIPTION
**Pull request by apella12**
This should close #5846 . There are no parameters and no association groups.

PR description here
[Qubino_Shades_Remote.pdf](https://github.com/zwave-js/node-zwave-js/files/12750157/Qubino_Shades_Remote.pdf)

memo: How this device appears to work 1) Includes in network (gets ID) 2) Includes Qubino blinds during operation.  Probably picks up zwave transmissions. 3) operate blinds with remote. Unclear how many blinds it can control.

### CODE REVIEW

**Comment by QubinoHelp** [comment]
Hi @apella12,

The Qubino Shades Remote controller has no parameters and it is included (added) to the gateway - hub as a controller type device.
The associations are done automatically by the remote depending on the device that is included in the remote. 
It can control up to 9 rollers or in case of blinds it can control 4 of them as one channel is used for up/down movement the second for the tilt, total 8 channels one remain free.
 
Link to pairing video: https://youtu.be/P4aeIGOV7aA
 
[Pairing the Qubino Shades Remote Controller.pdf](https://github.com/zwave-js/node-zwave-js/files/12762186/Pairing.the.Qubino.Shades.Remote.Controller.pdf)

File: packages/config/config/devices/0x015a/jtb-zmnkgd1.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+	"manufacturer": "Jin Tao Bao",
+	"manufacturerId": "0x015a",
+	"label": "JTB-ZMNKGD1",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015a/jtb-zmnkgd1.json:4]: @QubinoHelp I assume this is the manufacturer that produced this device for you?
We typically put the brand name in the `manufacturer` field, so should this be changed to Qubino, or is this one of those white-label devices that's sold under multiple brands?

Comment (QubinoHelp) [reply packages/config/config/devices/0x015a/jtb-zmnkgd1.json:4]: Yes they produce the remotes for us. I suppose it can be added under multiple brands.

Comment (AlCalzone) [reply packages/config/config/devices/0x015a/jtb-zmnkgd1.json:4]: Ok then we'll have to leave this as-is.

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**